cdap-app-fabric/src/main/java/co/cask/cdap/scheduler/CoreSchedulerService.java

```diff
@@ -186,6 +201,13 @@ public Void run(ProgramScheduleStoreDataset store) {
         return null;
       }
     }, RuntimeException.class);
+    List<String> namespaceIds = Lists.transform(namespaceMetas, new Function<NamespaceMeta, String>() {
+      @Override
+      public String apply(NamespaceMeta input) {
+        return input.getNamespaceId().toString();
+      }
+    });
+    LOG.info("Schedule migration is completed for all namespaces '{}'.", namespaceIds);
```

Comment:
It would be better to not log all the namespaceids here, since we are listing them one by one during migration.